test/e2e/e2e.go

```diff
@@ -79,8 +84,12 @@ func RunE2ETests(t *testing.T) {
 	RunSpecsWithDefaultAndCustomReporters(t, "Metallb Operator Validation Suite", rr)
 }
 
-var _ = Describe("validation", func() {
+var _ = Describe("metallb", func() {
```

Comment:
maybe we shouldn't change the suite name here as we are going to split it anyway

Reply:
When we split this we want to have "metallb" in both places. Without this change it will be hard to run focused tests in cnf.